api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreProperties.java

```diff
@@ -29,9 +29,14 @@ public void setApplicationContext(ApplicationContext applicationContext) {
         this.applicationContext = applicationContext;
     }
 
-    public Integer getOpenMrsIdentifierSourceId() {
+	public Integer getOpenMrsIdentifierSourceId() {
         String propertyName = RegistrationCoreConstants.GP_OPENMRS_IDENTIFIER_SOURCE_ID;
-        return getIntegerProperty(propertyName);
+		return getIntegerProperty(propertyName);
```

Comment:
I'm sure the above formatting is cause as a result of https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse .